ethereum_history_api/oracles/.eslintrc.cjs

```diff
@@ -16,5 +23,6 @@ module.exports = {
         ]
       }
     ]
-  }
+  },
+  overrides: [unrestrictedTemplateExpressionsInNoirFixturesScripts]
```

Comment:
I propose to fix Noir template script code (i.e. add `.join(',')` to arrays inside string template) that generate those errors instead of disabling this rule.